src/wasm.h

```diff
@@ -2198,6 +2198,7 @@ class Module {
   Global* getGlobal(Name name);
   Tag* getTag(Name name);
 
+  bool isExported(const Function& func) const;
```

Comment:
We generally try to keep `wasm.h` pretty minimal. I think a better place for this particular helper would be in `src/ir/export-utils.h`.